examples/rsp-cra-18/src/Completed.tsx

```diff
@@ -0,0 +1,38 @@
+import Delete from '@spectrum-icons/workflow/Delete';
+import {AlertDialog, DialogTrigger, ActionButton} from '@adobe/react-spectrum'
+import {Checkbox} from '@adobe/react-spectrum'
+import {Flex} from '@adobe/react-spectrum'
+
+
+function Completed(props: {completed: {id: number; task: string}[]; onDelete: any}){
```

Comment:
For readability, you may want to pull these types out into an interface, https://github.com/adobe/react-spectrum/blob/df1783a7f62ea20183aaf14ce549e6c1f80046e5/packages/%40react-spectrum/button/src/FieldButton.tsx#L23-L28 may be a good example